test/e2e/networkpolicy_test.go

```diff
@@ -270,6 +271,152 @@ func TestDefaultDenyEgressPolicy(t *testing.T) {
 	}
 }
 
+// TestEgressToServerInCIDRBlock is a duplicate of upstream test case "should allow egress access to server in CIDR block
+// [Feature:NetworkPolicy]", which is currently buggy in TOT release for clusters which use IPv6.
```

Comment:
s/TOT release/1.19 release

(after looking at the PR you shared, I think it's more appropriate)

Reply:
Updated.

Yes, not only alpha version image may lead to other issues, but also sonobuoy pulls images from gcr.io and `v1.20.0-alpha.0` is the latest, not `v1.20.0-alpha.2`. So I think this is the most efficient way to test these 2 cases.